File: LeetCode/Easy/2833-furthest-point-from-origin/2833-furthest-point-from-origin.py
class Solution:
    def furthestDistanceFromOrigin(self, moves: str) -> int:
        blank,l,r=0,0,0
        for ch in moves:
            if ch=='L' :
                l+=1
            elif ch =='R' :
                r+=1
            else :
                blank+=1
        return abs(l-r) + blank

File: LeetCode/Easy/2833-furthest-point-from-origin/2833-furthest-point-from-origin.cpp
class Solution {
public:
    int furthestDistanceFromOrigin(string moves) {
        int blank=0;
        int l=0;
        int r=0;
        for(int i=0;i<moves.length();i++){
            if(moves[i]=='L') l++;
            else if(moves[i]=='R') r++;
            else blank++;
        }
        return (abs(l-r) + blank);
    }
};